src/components/Map/provider.js

```diff
@@ -0,0 +1,27 @@
+import {get} from 'api';
+
+class OpenFormsProvider {
+  constructor(baseUrl) {
+    this.baseUrl = `${baseUrl}geo/address-search`;
+  }
+
+  parse(results) {
+    return results.map(location => ({
+      label: location.label,
+      x: location.latLng.lng,
+      y: location.latLng.lat,
+    }));
+  }
+
+  async search({query}) {
+    const searchUrl = this.baseUrl;
+    const searchResults = await get(searchUrl, {
```

Comment:
```suggestion
    const searchResults = await get(this.endpoint, {
```
and set the `this.endpoint` in the constructor.

There's no benefit to assigning an intermediate variable.